src/utils/constants.js

```diff
@@ -1,3 +1,3 @@
 const AUTH_STORAGE_KEY = 'wa_cert_authenticated';
-
-export { AUTH_STORAGE_KEY };
+const YT_BASE_URL = 'https://www.googleapis.com/youtube/v3/search';
```

Comment:
Why are these values not ENV variables? I believe they should be, at least the base url as we could have a different url for QA. The auth key could remain the same as this is probably shared between environments.